packages/@n8n/nodes-langchain/package.json

```diff
@@ -133,7 +133,7 @@
     "@aws-sdk/credential-provider-node": "3.549.0",
     "@getzep/zep-js": "0.9.0",
     "@google-ai/generativelanguage": "0.2.1",
```

Comment:
Could we please also soon upgrade `@google-ai/generativelanguage`. it's currently pulling in some vulnerable sub-dependencies.

Reply:
Unfortunately we don't have much control of this as it's a transitive dependency of Langchain :/

Reply:
😞